applications/nrf_desktop/doc/smp_custom.rst

```diff
@@ -0,0 +1,81 @@
+.. _nrf_desktop_smp_custom:
```

Comment:
Please use `_nrf_desktop_dfu_mcumgr` for consistency with documentation of our other modules (the event propagation array generation scripts also assume matching naming convention)

Reply:
I intentionally named it similarly to our `nrf_desktop_smp module` so that the user can be aware that these modules are very similar.

Changing the label should trigger the change in the module's name and the file's name. Are you sure that we want to do it?

Reply:
Discussed offline. Module will be renamed for consistency with the current convention.